gnome-initial-setup/pages/endless-eula/gis-endless-eula-viewer.c

```diff
@@ -142,14 +146,15 @@ get_terms_document (void)
    */
   language = setlocale (LC_MESSAGES, NULL);
   locale_variants = g_get_locale_variants (language);
-  path = find_terms_document_for_languages ((const gchar * const *)locale_variants);
-  g_strfreev (locale_variants);
+  driver = GIS_DRIVER (g_application_get_default ());
 
+  if (gis_driver_is_hack (driver))
+    path = find_terms_document_for_languages ("Hack-Terms-of-Use.pdf",
```

Comment:
Please call it `Endless-Hack-Terms-of-Use.pdf`. However as things are today, this won't work, since we don't have a separate hack terms of use file. Please also open a PR for eos-license-service to create a symlink at least for now.

Reply:
It will fallback to the regular `Endless-Terms-of-Use.pdf` file.

Reply:
@ramcq Well, my original approach wasn't hack specific since it played with the `vendor.conf` file parameters instead of deciding stuff based on the value of the product name, `g-i-s` is already too hack specific from the point where we added the `gis_driver_is_hack` function. I don't see how this case is different.

So, I'll change it to be: `$datadir/eos-license-service/terms/$product/Terms-of-Use-$locale.pdf`